lms/templates/edxnotes/edxnotes.html

```diff
@@ -0,0 +1,93 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%! import json %>
+<%namespace name='static' file='/static_content.html'/>
+<%inherit file="/main.html" />
+
+<%block name="pagetitle">${_("Student Notes")}</%block>
+<%block name="headextra">
+    <%static:css group='style-course'/>
+</%block>
+
+<%include file="/courseware/course_navigation.html" args="active_page='edxnotes'" />
+<section class="container wrapper-student-notes">
+    <div class="info-wrapper">
+        <section class="updates">
+            <div class="title-search-container">
+                <div class="wrapper-title">
+                    <h1 class="page-title">${_('Notes')} <small class="page-subtitle">${_("Highlights and notes you've made in course content")}</small></h1>
+                </div>
+            % if notes:
+                <div class="wrapper-notes-search">
+                    <form role="search" action="${search_endpoint}" method="GET" id="search-notes-form" class="is-hidden">
+                        <label for="search-notes-input" class="sr">${_('Search notes for:')}</label>
+                        <input type="search" class="search-notes-input" id="search-notes-input" name="note" placeholder="${_('Search notes for...')}">
+                        <button type="submit" class="search-notes-submit"><i class="icon-search"></i> ${_('Search')}</button>
+                    </form>
+                </div>
+            % endif
+            </div>
+
+            <section class="wrapper-tabs">
+                <div class="tab-list is-hidden">
+                    <h2 id="tab-view" class="tabs-label">${_('View notes by:')}</h2>
+                </div>
+                <div class="inline-error is-hidden" tabindex="-1"></div>
+            % if notes:
+                <div class="ui-loading" tabindex="-1">
+                    <p><span class="spin"><i class="icon-refresh"></i></span> <span class="copy">${_("Loading...")}</span></p>
+                </div>
+            % else:
+                <section class="is-empty">
+                    <article>
```

Comment:
The text in this and the following lines should match the text in the [Google doc](https://docs.google.com/a/edx.org/document/d/17NZrkqSVukAioy55KNPSetwex3ygxWfrJSCa_LHwP2w/edit#heading=h.sqkiilerf0p) - scroll down to the blue "Revisions" text.

You have not made any notes in this course yet.
Get started by making a note in something you just read, like {section_link}.
Other students in this course are using notes to:
- Mark a passage or concept so that it's easy to find later.
- Record thoughts about a specific passage or concept.
- Highlight important information to review later in the course or in future courses.